### PR TITLE
DSD-2272: expect string not boolean in template file.

### PR DESCRIPTION
**Pull request by creynolds15**

### CODE REVIEW

**Comment by tamr-github-integrator** [comment]
_No problems detected!_

**Comment by tamr-github-integrator** [comment]
_No problems detected!_

**Comment by creynolds15** [comment]
@souza-dan Looks like terraform `templatefile` function errors out doing a boolean check on strings. Updated to correctly check for empty string.

**Comment by tamr-github-integrator** [comment]
_No problems detected!_

File: templates/startup_script.sh.tmpl [hunk 1]
```diff
@@ -17,7 +17,7 @@ EOF
 # check if tamr has been installed on the VM, if not download and install it
 if [[ ! -f ${tamr_home_directory}/tamr/start-unify.sh ]]; then
     # run any custom commands to prepare VM
-    %{ if pre_install_bash }
+    %{ if pre_install_bash != ""}
```
Comment (souza-dan) [inline templates/startup_script.sh.tmpl:20]: Do we need the check at all? If `pre_install_bash` is empty and we rendered it, wouldn't it just render an empty line?

Comment (creynolds15) [reply templates/startup_script.sh.tmpl:20]: Yes that is true. The check is trivial and not needed.

**Comment by tamr-github-integrator** [comment]
_No problems detected!_

**Review by souza-dan — APPROVED** [review]
LGTM

**Pull request merged**